static/coffee/locations.coffee

```diff
@@ -2,6 +2,7 @@ initMap = (id, geojson, question, districtLabel) ->
   map = L.map(id, {scrollWheelZoom: false, zoomControl: false, touchZoom: false, trackResize: true,  dragging: false}).setView([0, 0], 8)
 
   allowDistrictZoom = districtLabel.trim() != ''
+  nullDataCount = 0
```

Comment:
I have some doubts about how `nullDataCount` is used. If it is increased just once, why can't you just use `data` directly? And if it is increased more than once, why do you compare it with `1` at line `209`?